source/hotkeys_.d

```diff
@@ -209,8 +225,8 @@ struct ModHotkey{
 
 struct Hotkeys{
 	bool capsIsCtrl=true;
-	int[] moveForward,moveBackward,turnLeft,turnRight;
-	int[] cameraZoomIn,cameraZoomOut;
+	int[] moveForward,moveBackward,turnLeft,turnRight, moveUp, moveDown;
+	int[] cameraZoomIn,cameraZoomOut, cameraUp, cameraDown, cameraLeft, cameraRight, cameraForward, cameraBackward;
```

Comment:
Likewise I would prefer those on their own line, between `moveUp,moveDown` and the camera zoom hotkeys. If you don't feel like addressing those really minor nitpicks I can also do a fixup commit after I merge.

Reply:
changed as per what we discussed offline